app/globals.css

```diff
@@ -1,92 +1,113 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
- 
+
```

Comment:
I'm experiencing some mild visual bugs on that latest deploy.
![Peek 2024-03-02 17-49](https://github.com/serotracker/dashboard/assets/86806388/61e7a7ac-f587-400d-8c3b-d750f026f3bc)

Reply:
Yeah there was a merge issue that deleted the styles, just added them back in